### PR TITLE
chore(security):[TRACEFOSS-XXX] fixed high security issues

### PR DESCRIPTION
**Pull request by ds-mmaul**

### CODE REVIEW

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnInvestigationsPageable.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.eclipse.tractusx.traceability.assets.application.rest.request;
```
Comment (ds-mwesener) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnInvestigationsPageable.java:1]: Should be within a request package from InvestigationsController

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnRegistryLookUpMetricsPageable.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package org.eclipse.tractusx.traceability.assets.application.rest.request;
```
Comment (ds-mwesener) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnRegistryLookUpMetricsPageable.java:1]: Should be within a request package from RegistryLookupMetricController

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnInvestigationsPageable.java (hunk 1 above)
Comment (ds-mwesener) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnInvestigationsPageable.java:1]: Please delete

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnRegistryLookUpMetricsPageable.java (hunk 2 above)
Comment (ds-mwesener) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnRegistryLookUpMetricsPageable.java:1]: Please delete

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnAssetsPageable.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package org.eclipse.tractusx.traceability.assets.application.rest.request;
```
Comment (ds-mwesener) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/application/rest/request/OwnAssetsPageable.java:1]: Call it OwnPageable and move it to common/request/ package and reuse for InvestigationsController and RegistryLookupMetricsController as well.

File: CHANGELOG.md [hunk 4]
```diff
@@ -70,6 +70,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added Dockerfile to root directory
```
Comment (ds-mwesener) [inline CHANGELOG.md:70]: After refactoring please address integration tests failing due to request model change.

Comment (ds-mmaul) [reply CHANGELOG.md:70]: Adressed, tests and check now seem to run through

**Pull request merged**